pkgs/tools/misc/vector/default.nix

```diff
@@ -1,42 +1,53 @@
-{ stdenv, lib, fetchFromGitHub, rustPlatform
-, openssl, pkg-config, protobuf
-, Security, libiconv, rdkafka
-, tzdata
+{ stdenv, lib, fetchFromGitHub, rustPlatform, openssl, pkg-config, protobuf
+, Security, libiconv, rdkafka, tzdata
 
-, features ?
-    ((if stdenv.isAarch64
-     then [ "jemallocator" "rdkafka" "rdkafka/dynamic_linking" ]
-     else [ "leveldb" "leveldb/leveldb-sys-2" "jemallocator" "rdkafka" "rdkafka/dynamic_linking" ])
-     ++
-     (lib.optional stdenv.targetPlatform.isUnix "unix")
-     ++
-     [ "sinks" "sources" "transforms" ])
-, coreutils
-, CoreServices
-}:
+, features ? ((if stdenv.isAarch64 then [
+  "jemallocator"
+  "rdkafka"
+  "rdkafka/dynamic_linking"
+] else [
+  "leveldb"
+  "leveldb/leveldb-sys-2"
+  "jemallocator"
+  "rdkafka"
+  "rdkafka/dynamic_linking"
+]) ++ (lib.optional stdenv.targetPlatform.isUnix "unix")
+  ++ [ "sinks" "sources" "transforms" ]), coreutils, CoreServices }:
 
 rustPlatform.buildRustPackage rec {
   pname = "vector";
-  version = "0.10.0";
+  version = "0.11.0";
 
   src = fetchFromGitHub {
-    owner  = "timberio";
-    repo   = pname;
-    rev    = "v${version}";
-    sha256 = "0q6x3fvwwh18iyznqlr09n3zppzgw9jaz973s8haz54hnxj16wx0";
+    owner = "timberio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "xyk/6WYCzD1IM/jgt+6VT2GTNClE8qdUSEXWQlpY/Bg=";
   };
 
-  cargoSha256 = "Y/vDYXWQ65zZ86vTwP4aCZYCMZuqbz6tpfv4uRkFAzc=";
+  cargoSha256 = "";
```

Comment:
You need to set this.

Reply:
Agreed. I wanted to make it clear that this needs to be updated. But you are perfectly right!
I'll leave this here until the build issue has been solved.
Thanks!